apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/config/GrpcConfig.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.api.common.config;
+
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+import java.time.Duration;
+import java.util.Optional;
+
+/** Configuration for the gRPC calls to the Bridge. */
+@ConfigMapping(prefix = "stargate.grpc")
+public interface GrpcConfig {
+
+  /** @return Optional deadline duration for the each RPC to the bridge. Defaults to 30 seconds. */
+  @WithDefault("PT30S")
```

Comment:
Isn't 30 seconds kind of high?
I guess it is lower than what the ultimate default would be (or maybe there was no deadline specified at all).

Would think that something like 10 seconds might be better but maybe we might have some data to suggest something less than 30 seconds that is rarely seen.

Reply:
* 30 seconds is what we agreed to be the value for now
* is it too much, maybe, but
   * better that not having at all (yes it's unlimited)
   * or to get timeout from the sidecar (60 sec default on nginx)
* and value is configurable, so you can set whatever you want when starting

Reply:
Ok.